### PR TITLE
update to 3.3.2

### LINKED ISSUE #10: scipy 1.0 breaks current recipe
I'll work on rolling out a new pyamg release that fixes `calc_work`
https://github.com/pyamg/pyamg/issues/213

### PR DESCRIPTION
**Pull request by lukeolson**
This fixes https://github.com/conda-forge/pyamg-feedstock/issues/10.

### CODE REVIEW

**Comment by lukeolson** [comment]
@jakevdp this looks ready now

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
-{% set version = "3.2.1" %}
+{% set name = "pyamg" %}
+{% set version = "3.3.2" %}
 
 package:
     name: pyamg
```
Comment (jakevdp) [inline recipe/meta.yaml]: Probably not a big deal, but could use ``{{ name }}`` here

**Comment by jakevdp** [comment]
Looks good! I'm happy to merge as-is – let me know if you'd like to change the ``nam``/``{ name }`` thing.

**Comment by jakevdp** [comment]
Thanks!

**Pull request merged**